src/organisms/cards/PlaybackCard/index.js

```diff
@@ -14,10 +15,11 @@ function PlaybackCard(props) {
     onClick,
     icon,
     editLinkText,
+    marginBottom,
   } = props;
 
   return (
-    <div className={styles['playback-card']}>
+    <div className={classnames(styles['playback-card'], marginBottom && styles['add-margin'])}>
```

Comment:
@cisacke Components should not have their own margins -- they should be positioned by their parents. In the future, it will be hard / complicated to reposition this Playback component in a different context.